ecommerce/baskets/models.py

```diff
@@ -0,0 +1,41 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from core.models import BaseAbstractModel
+from baskets import enums
+from customers.models import Customer
+from products.models import Product
+
+currency = 'TL'
```

Comment:
Currency seems hard-coded. There might be a platform which supports multi-currency orders so this approach might ignore other currencies on the fly.

Reply:
I meant to try if its works or not. It seems I forgot about it thank you.